incubator/kolide-fleet/values.yaml

```diff
@@ -0,0 +1,83 @@
+---
+## Change the name of the resource
+## Default: use the chart name
+# nameOverride:
+
+mysql:
+  mysqlUser: "kolide"
+  mysqlDatabase: "kolide"
+  ## (Required)
+  mysqlPassword: ""
+
+redis:
+  ## (Required)
+  redisPassword: ""
+
+## Fleet application specific settings
+fleet:
+  mysql: {}
+    # address: ""
+    # database: ""
+    # tls_cert: ""
+    # tls_key: ""
+    # tls_ca: ""
+    # tls_server_name: ""
+    # tls_config: ""
+  server: {}
+    # tls: ""
+    # tlsprofile: "modern"
+  auth:
+    ## (Required)
+    jwt_key: ""
+    # bcrypt_cost: ""
+    # salt_key_size: ""
+  app: {}
+    # token_key_size: ""
+    # invite_token_validity_period: ""
+  session: {}
+    # key_size: ""
+    # duration: ""
+  osquery: {}
+    # node_key_size: ""
+    # label_update_interval: ""
+  logging:
+    ## Whether to enable debug logging
+    debug: false
+    ## Whether logging should be expressed in JSON
+    json: true
+    ## Whether to disable the banner as part of the logs
+    disable_banner: true
+
```

Comment:
Nesting by `mysql`, `redis`, and `fleet` makes sense. Move `pod`, `service`, and `resources` under `fleet`. The `image` and `replicas` should be directly under `fleet` then.